### ISSUE TITLE
Design review: Jump to

### ISSUE THREAD

**Issue by churik** [status-im/status-react#16836]
- [ ] 1. Add blur and opacity for close icon (IOS, Android) - comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=3662:32691&mode=design#510054665)  - tracked as separate issue in https://github.com/status-im/status-mobile/issues/16276

- [ ] 2. Placeholder is used instead of the actual image if the last message in 1-1 / group chat is a picture

https://github.com/status-im/status-mobile/assets/4557972/f153529a-5bb8-4baa-b540-a2cc0428cc11
Blocked by status-go bug: https://github.com/status-im/status-mobile/issues/14625

- [x] 3. No fallback letter on channel on the community card (fixed in https://github.com/status-im/status-mobile/pull/16863)

![IMAGE 2023-08-02 12:31:54](https://github.com/status-im/status-mobile/assets/4557972/a70c71f4-ebfe-4ebb-b84c-15708339a9e1) (should be G)

**Comment by J-Son89** [comment]
@churik most likely we can close this issue - @codemaster115 is working on https://github.com/status-im/status-mobile/issues/16276 which should address this correctly 👍

**Comment by Parveshdhull** [comment]
Hi @churik, Thank you very much for creating issue.
About issue 2, we are using placeholder because status-go is not returning urls for image. logged in https://github.com/status-im/status-mobile/issues/14625

**Comment by Parveshdhull** [comment]
About issue 3, tracked here https://github.com/status-im/status-mobile/issues/16577

**Comment by Parveshdhull** [comment]
> About issue 3, tracked here #16577

Looks like we don't use context-tag for community channel cards, we just use `channel-avatar` component. So its different issue, we need to fix it.

**Comment by ilmotta** [comment]
Closing as _not planned_ because Jump To feature is descoped and disabled by a feature flag.
